### PR TITLE
Continuos deployment added

### PR DESCRIPTION
**Pull request by JoseDRojasA**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+frontend
+backend/node_modules
```
Comment (JuanDiegoLopez) [inline .dockerignore]: Please add new line at the end of the line

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+        run: heroku container:login 
+      - name: Build and push
+        env:
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+        run: heroku container:push -a ${{ secrets.HEROKU_APP_NAME }} web 
+      - name: Release
+        env:
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+        run: heroku container:release -a ${{ secrets.HEROKU_APP_NAME }} web
```
Comment (JuanDiegoLopez) [inline .github/workflows/workflow.yml]: Same here

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ENV DB_USERNAME=_DB_USERNAME_
+ENV DB_PASSWORD=_DB_PASSWORD_
+ENV DB_NAME=_DB_NAME_
+ENV API_PORT=_API_PORT_
+
+COPY backend/package*.json ./
+RUN npm install
+COPY backend/ ./
+EXPOSE _API_PORT_
+CMD [ "node", "index.js" ]
```
Comment (JuanDiegoLopez) [inline Dockerfile]: Same here

File: .github/workflows/workflow.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: Deployment to Heroku
+
+on:
+  push:
+    branches:
+      - master
```
Comment (JuanDiegoLopez) [inline .github/workflows/workflow.yml]: Should it not be develop branch?

Comment (JoseDRojasA) [reply .github/workflows/workflow.yml]: Yeah, You're right, thanks

**Review by JuanDiegoLopez — APPROVED** [review]
Looks good! Approved

**Pull request merged**